test/jdk/java/nio/channels/DatagramChannel/SelectWhenRefused.java

```diff
@@ -25,14 +25,21 @@
  * @bug 6935563 7044870
  * @summary Test that Selector does not select an unconnected DatagramChannel when
  *    ICMP port unreachable received
+ * @library /test/lib
+ * @run main/othervm SelectWhenRefused
  */
 
 import java.nio.ByteBuffer;
 import java.nio.channels.*;
 import java.net.*;
 import java.io.IOException;
+import java.util.Set;
+
+import static jdk.test.lib.Asserts.assertNotEquals;
```

Comment:
Ah! Darn. I wonder if we should convert the test to testng or junit instead, just for the purpose of using junit/testng assertions. It would avoid the need of linking to /test/lib too. Just some idle thought.
The other day Pavel and I were wondering if that Asserts class should be retired - given that we have them available in testng/junit now...

Reply:
I wouldn't be against changing it, At a glance using Junit is only around 10 lines of changes, though I'm not sure if theres any additional patterns I should follow in swapping it to junit beyond using `@Test` instead of a main method?

Reply:
I made the change there and made sure the test was still working as intended